views/extras.tmpl

```diff
@@ -225,22 +227,36 @@
             $("#decode_header").text("Sent tx");
             $("#decoded_tx").text(evt);
         })
+
         var updateBlockData = function (event) {
             console.log("Received newblock message", event);
             var newBlock = JSON.parse(event);
-            var confirms = $('[data-confirmation-block-height]');
-            $.each(confirms, function(i, v) {
-                if (isNaN($(v).text())) {
-                    $(v).text("0");
-                }
-                var confirmations = newBlock.block.height - $(v).data('confirmation-block-height');
-                $(v).text("(" + confirmations + (confirmations > 1? " confirmations": " confirmation") + ")")
-            })
 
             // block summary data
             var b = newBlock.block;
             desktopNotifyNewBlock(b);
 
+            var currentTxHash = $('#spanTxID').data('tx-page-txid');
+            if(currentTxHash) { // we are on the tx page
+                Turbolinks.reload()
+                /*if((b.Tx.filter(function (tx) {
+                    return tx.TxID === currentTxHash
+                })).length > 0){
+                    // the current tx is in the received block
+                    Turbolinks.visit(window.location)
+                }*/
```

Comment:
Does this comment have value? (honest question)

Reply:
> Does this comment have value? (honest question)

The essence was to check that the current tx id is contained in the received block before updating the view.
What I noticed is that there were some block received events (from the websocket) that were updating the confirmation counts wrongly.

The number becomes high but when you do a full page reload you see a different number (the actual count).
I attributed this to the fact that the current transaction was not present in the received block but commented it out since I am now silently reloading the page and not doing a partial update.

Reply:
That makes sense.  TBH, it would be nice to have like 2 sentences to that extent in the vicinity of the comment.  Such comments are helpful to prevent similar gotchas in the future, and justify the fill silent reload rather than partial update.